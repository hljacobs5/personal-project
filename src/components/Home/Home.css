.home-screen {  
  /*background-color: #B8D1E1;*/
  /*min-height: 100vh;*/
 }

 h4 {
font-family: 'Kodchasan', sans-serif;
 	font-size: 3.5rem;
 	color: #B8D1E1;
 }

 h6 {
 	color: #B8D1E1;
font-family: 'Kodchasan', sans-serif;
font-size: 2rem;
 }
