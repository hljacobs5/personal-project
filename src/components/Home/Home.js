import React from 'react';
import './Home.css';

const Home = () => {
	return (
		<div>
		  <h4>Educate yourself about your passion.</h4>
		</div>
	)
}

export default Home;