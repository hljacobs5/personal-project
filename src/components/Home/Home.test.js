import React from 'react';
import { shallow, mount } from 'enzyme';
import { Home } from './Home'

describe('Home', () => {
	let wrapper;

	it('should exist', () => {
		expect(wrapper).toBeDefined();
	})

	it('should render like snapshot', () => {
		expect(wrapper).toMatchSnapshot();
	})
})