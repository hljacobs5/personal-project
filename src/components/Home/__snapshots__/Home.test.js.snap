// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render like snapshot 1`] = `undefined`;
